src/ContactPage/Contact.scss

```diff
@@ -0,0 +1,68 @@
+@import "../styles/typography";
+
+.contact-section {
+  height: 100vh;
+
+  &__border {
+    border: none;
+    padding: 2% 0;
+    background: rgb(229, 226, 226);
+  }
+  &__title {
+    font-family: $base-font-family;
+    margin: 0 5%;
+    margin-bottom: 10px;
+  }
+
+  &__form {
+    margin: 0 5%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__submit {
+    background-color: rgb(238, 59, 74);
+    border-radius: 5px;
+    color: white;
+    border: none;
+    text-align: center;
+    text-decoration: none;
+    font-size: 1em;
+    width: 120px;
+    height: 30px;
+    margin-top: 10px;
+
+    &:disabled {
+      background-color: grey;
```

Comment:
I dont think making this grey is that clear its disabled. Especially on a grey background